core/domain/taskqueue_services.py

```diff
@@ -0,0 +1,118 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Provides a seam for taskqueue-related operations."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import datetime
+import feconf
+import json
+import python_utils
+from core.platform import models
+
+platform_taskqueue_services = models.Registry.import_taskqueue_services()
+
+
+# NOTE: The following constants should match the queue names in queue.yaml.
```

Comment:
Add a backend test to verify -- it can probably extract the names from queue.yaml by regex and ensure that each of those matches some constant in this file.